src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf

```diff
@@ -100,7 +100,7 @@
             </trans-unit>
             <trans-unit id="25">
                 <source>This value is not valid.</source>
-                <target>Deze waarde is ongeldig.</target>
+                <target>Deze waarde is niet geldig.</target>
```

Comment:
Eventually we can translate it in many ways, i chose for consistent syntax. Imho this reads as well/friendly/fluent given the context of these translations. I dont understand whats wrong with translating `not valid` to `niet geldig`.